rir/src/interpreter/interp.c

```diff
@@ -494,27 +480,22 @@ INLINE SEXP rirCallClosure(SEXP call, SEXP env, SEXP callee, SEXP actuals,
 
     ostack_push(ctx, newEnv);
 
-    // Create a new R context and store it on the stack
```

Comment:
@o- I don't understand this part at all (not criticism, just statement of fact). Why can this be so drastically simplified?